dnanexus/build_seaseaq/src/build_seaseaq.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+# build_seaseaq 0.0.1
+# Generated by dx-app-wizard.
+#
+# Basic execution pattern: Your app will run on a single machine from
+# beginning to end.
+#
+# Your job's input variables (if any) will be loaded as environment
+# variables before this script runs.  Any array inputs will be loaded
+# as bash arrays.
+#
+# Any code outside of main() (or any entry point you may add) is
+# ALWAYS executed, followed by running the entry point itself.
+#
+# See https://documentation.dnanexus.com/developer for tutorials on how
+# to modify this file.
+
+main() {
+
+    # Fill in your application code here.
+    #
+    # To report any recognized errors in the correct format in
+    # $HOME/job_error.json and exit this script, you can use the
+    # dx-jobutil-report-error utility as follows:
+    #
+    #   dx-jobutil-report-error "My error message"
+    #
+    # Note however that this entire bash script is executed with -e
+    # when running in the cloud, so any line which returns a nonzero
+    # exit code will prematurely exit the script; if no error was
+    # reported in the job_error.json file, then the failure reason
+    # will be AppInternalError with a generic error message.
+    git clone https://github.com/stjude/seaseq.git seaseq
+    cd seaseq
+    git checkout 1.0
+    cd dnanexus
+    dx mkdir -p ${DX_PROJECT_CONTEXT_ID}:/apps/
+    reorg_id=$(dx build -d ${DX_PROJECT_CONTEXT_ID}:/apps/ -f seaseq-reorg | jq -r '.id')
+    echo "Reorg applet ID: ${reorg_id}"
+    sed -ibak "s/applet-Fx40j6091FfQp1P99p6b5k2x/${reorg_id}/" extras.json
+    cd ..
+    wget https://github.com/dnanexus/dxWDL/releases/download/v1.50/dxWDL-v1.50.jar
+    echo '476621564b3b310b17598ee1f02a1865 dxWDL-v1.50.jar' > dxWDL-v1.50.jar.md5
+    md5sum -c dxWDL-v1.50.jar.md5
+    # Test without the extras since running this way doesn't upload outputs to the project.
+    # -extras dnanexus/extras.json 
```

Comment:
`extras.json` or `seaseq-reorg` applet is a required part of the pipeline. It works as it should if I run the seaseq pipeline directly rather than from the `build_seaseq` applet.